pkg/client/leaderelection/leaderelection.go

```diff
@@ -296,3 +312,49 @@ func (le *LeaderElector) tryAcquireOrRenew() bool {
 	le.observedTime = time.Now()
 	return true
 }
+
+func DefaultLeaderElectionCLIConfig() LeaderElectionCLIConfig {
+	return LeaderElectionCLIConfig{
+		LeaderElect:   false,
+		LeaseDuration: DefaultLeaseDuration,
+		RenewDeadline: DefaultRenewDeadline,
+		RetryPeriod:   DefaultRetryPeriod,
+	}
+}
+
+// LeaderElectionCLIConfig is useful for embedding into component configuration objects
+// to maintain consistent command line flags.
+type LeaderElectionCLIConfig struct {
+	LeaderElect   bool
+	LeaseDuration time.Duration
+	RenewDeadline time.Duration
+	RetryPeriod   time.Duration
+}
+
+// BindFlags binds the common LeaderElectionCLIConfig flags to a flagset
+func (l *LeaderElectionCLIConfig) BindFlags(fs *pflag.FlagSet) {
+	clean := func(s string) string {
```

Comment:
What is the purpose of this? Is it just so you can split each string across multiple lines? Why not just put it in one (very long) line?